tests/components/textfield.test.js

```diff
@@ -0,0 +1,170 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import TextField from "@libs/components/textfield";
+
+describe("components/TextField", () => {
+  it("can have a label", () => {
+    const component = renderer.create(
+    	<TextField 
+    	  id="1"
+    	  label="foo" 
+    	/>
+   	);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("can have a type", () => {
+    const component = renderer.create(
+    	<TextField 
+    	  id="1"
+    	  type="bar" 
+    	/>
+   	);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  
+  it("can be disabled", () => {
+    const component = renderer.create(
+    	<TextField 
+    	  id="1"
+    	  disabled
+    	/>
+   	);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("can be dense", () => {
+    const component = renderer.create(
+    	<TextField 
+    	  id="1"
+    	  dense
+    	/>
+   	);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("can have an helperText", () => {
+    const component = renderer.create(
+    	<TextField 
+    	  id="1"
+    	  helperText="foobar"
+    	/>
+   	);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("can be htPersistent", () => {
```

Comment:
what does that mean? this idea here, in case you did not get it, is to put meaningful information for our users.

Reply:
I don't know what htPersistant does so I can't make this :)

Reply:
that's why we need tests :) if it does mean anything to you, then what do you think it means for me or other people? In that case, let's ask @mikbry and maybe see if we can rename the prop to something more meaningful.